concepts/05 Widgets/Chart/11 Series Types/10 Line Series.md

```diff
@@ -68,6 +68,73 @@ To specify one or another series type, assign its name to the **series[]**.[type
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ... >
+            <DxSeries type="spline"/>
+            <DxSeries ... />
+            ...
+            <DxCommonSeriesSettings
+                :line=" ... "
+                :spline=" ... "
+                :stepline=" ... "
+                :stackedline=" ... "
+                :stackedspline=" ... "
+                :fullstackedline=" ... "
+                :fullstackedspline=" ... "
```

Comment:
It is better to declare the objects (`lineSettings`, `stackedSplineSettings`, etc.) in `data()` and bind the attributes to them. The same thing in React and in other similar cases in this PR.

Reply:
Fixed